pkg/ebpf/c/common/context.h

```diff
@@ -128,6 +128,8 @@ statfunc int init_program_data(program_data_t *p, void *ctx, u32 event_id)
 
     p->event->args_buf.offset = 0;
     p->event->args_buf.argnum = 0;
+    __builtin_memset(p->event->args_buf.args_offset, 0xFF, sizeof(p->event->args_buf.args_offset));
```

Comment:
We use `0xFF` as the initial value for the `args_offset` buffer to mark all index values as invalid. Using a different value (e.g., 0x00 or 0x80) could lead to confusion, as it might be interpreted as a valid value.

This behavior is verified in the following logic, where INVALID_ARG_OFFSET is defined as `0xFFFF`:

```c
    if (offset == INVALID_ARG_OFFSET)
        return 0;
```